### PR TITLE
use audio files from p5.js website assets folder

### PR DESCRIPTION
**Pull request by ogbabydiesal**
current main branch uses audio assets from [tone](https://tonejs.github.io/), these diffs update the audio files to use the old assets in the p5.js assets folder - would be great to update these sounds to new ones very soon, maybe keep a few of the old ones.

### CODE REVIEW

File: src/Amplitude.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { Context as ToneContext } from "tone/build/esm/core/context/Context.js";
  *   
  * function preload() {
  *   //replace this sound with something local with rights to distribute
- *   sound = loadSound('https://tonejs.github.io/audio/berklee/gong_1.mp3');
+ *   sound = loadSound('assets/Damscray_DancingTiger.mp3');
```
Comment (limzykenneth) [inline src/Amplitude.js]: For the assets to be linked correctly on the website, these needs to link from root.
```suggestion
 *   sound = loadSound('/assets/Damscray_DancingTiger.mp3');
```

**Review by limzykenneth — CHANGES_REQUESTED** [review]
One minor change to get the link working correctly on the website. We can merge after this.

**Review by ogbabydiesal — COMMENTED** [review]
added '/' for filepaths

**Comment by limzykenneth** [comment]
Looks good. Thanks!

**Pull request merged**